openrtb_ext/multibid.go

```diff
@@ -0,0 +1,89 @@
+package openrtb_ext
+
+import "fmt"
+
+const DefaultBidLimit = 1
+const MaxBidLimit = 9
+
+func ValidateAndBuildExtMultiBid(prebid *ExtRequestPrebid) ([]*ExtMultiBid, []error) {
+	if prebid == nil || prebid.MultiBid == nil {
+		return nil, nil
+	}
+
+	var validationErrs, errs []error
+	var validatedMultiBids, newMultiBids []*ExtMultiBid //returning slice instead of map to keep the downstream req.Ext payload consistent
+	multibidMap := make(map[string]struct{})            // map is needed temporarily for validate of duplicate entries, etc.
```

Comment:
Nitpick: can this be renamed to `multiBidMap` and I think this comment can be removed.

Reply:
done